pallets/balances/src/benchmarking.rs

```diff
@@ -20,32 +20,77 @@
 #![cfg(feature = "runtime-benchmarks")]
 
 use super::*;
+use crate::Module as Balances;
+use polymesh_common_utilities::benchs::UserBuilder;
 
-use frame_benchmarking::{account, benchmarks, whitelisted_caller};
+use frame_benchmarking::benchmarks;
 use frame_system::RawOrigin;
-// use polymesh_primitives::{CddId, Claim, IdentityId, InvestorUid};
-
-use crate::Module as Balances;
 
-const SEED: u32 = 0;
+fn make_worst_memo() -> Option<Memo> {
+    Some(Memo([7u8; 32]))
+}
 
 benchmarks! {
     _ { }
 
     // Benchmark `transfer` extrinsic with the worst possible conditions:
     // * Transfer will create the recipient account.
     transfer {
-        let caller = whitelisted_caller();
-        // Give some balance to the caller
-        let transfer_amount = T::Balance::from(500);
-        let _ = <Balances<T> as Currency<_>>::make_free_balance_be(&caller, transfer_amount*2.into());
-        // Give a valid CDD to the receiver
-        let recipient: T::AccountId = account("recipient", 0, SEED);
-        let recipient_lookup: <T::Lookup as StaticLookup>::Source = T::Lookup::unlookup(recipient.clone());
-        T::Identity::create_did_with_cdd(recipient.clone());
-    }: transfer(RawOrigin::Signed(caller.clone()), recipient_lookup, transfer_amount)
+        let amount = T::Balance::from(500);
+        let caller = UserBuilder::<T>::default().balance(1200).generate_did().build("caller");
```

Comment:
This module could use `user("caller", 1200)` to reduce noise.

Reply:
ditto